pkg/agent/controller/agent.go

```diff
@@ -81,6 +81,9 @@ func New(spec *AgentSpecification, syncerConf broker.SyncerConfig, kubeClientSet
 		return nil, errors.Wrap(err, "error converting resource")
 	}
 
+	waitForCacheSyncLocal := true
```

Comment:
Confirming:
this settings are going to apply to all scenarios?
If the broker is not set to `true` I would expect that the default value would suffice unless you want to make this explicit (in which case `waitForCacheSyncBroker` and `waitForCacheSyncLocal` suggest the same behavior and you may want to call them, e.g.,  `waitForCacheSyncLocal` and `dontWaitForCacheSyncBroker`)